docs/getting-started/application-setup/03-project-setup-homebrew-vs-nix/README.md

```diff
@@ -0,0 +1,28 @@
+---
+pagination_next: null
+---
+
+# Project Setup: Homebrew vs. Nix
+
+At this point, there are two main ways you can choose to continue setting up your local environment:
+
+- Using `nix` with a bit of `homebrew`
+- Using primarily only `homebrew`
+
+Nix and Homebrew are both package managers, but they have different philosophies and approaches to managing software and dependencies.
+
+You can check out the [Nix website](https://nixos.org/) for more information on Nix, but in essence it is a cross-platform package manager that is designed to provide a reproducible and isolated environment for software packages and their dependencies, which allows for easy versioning and rollback of packages.
+
+On the other hand, Homebrew focuses on providing a simple and user-friendly way to install and manage software packages. It doesn't enforce immutability or isolation to the same degree as Nix. Packages are installed to a common location, and the user is responsible for managing dependencies and potential conflicts.
+
+## Which Should I Choose?
+
+Ultimately, it’s up to you!
+
+Nix can be a great option, but it’s notoriously difficult to set up correctly as the [reference manual](https://nixos.org/manual/nix/stable/) is pretty dense, and just as difficult to uninstall if you decide not to use it later. But apparently, if set up correctly, it works *really* well.
```

Comment:
I don't know if I'd malign nix here, lol. Maybe something like

> Nix is known to have a steep learning curve and can be difficult to set up for first-time users. It also has less documentation available than other tools. Once working, it is usually a fairly seamless experience.

Reply:
Maybe I'm just a little biased from my own experiences with nix 😅 Updated!

Reply:
❌ 
Thanks, this felt a bit biased, but I was willing to look past it in my review. I'm incorporating both your re-write and my own here.

```suggestion
Nix is known to have a steep learning curve and can be difficult to set up for first-time users. It has [low-level documentation available](https://nixos.org/manual/nix/stable) compared to other tools. Once working, it is usually a fairly seamless experience.
```